### PR TITLE
added buildUnsafe() and loggingOptions

### PR DESCRIPTION
**Pull request by rjharmon**
- needs https://github.com/HeliosLang/uplc/pull/2
- needs https://github.com/HeliosLang/ledger-babbage/pull/2

Not needed:
- https://github.com/HeliosLang/compiler-utils/pull/3

### CODE REVIEW

File: src/txbuilder/TxBuilder.js [hunk 1]
```diff
@@ -39,10 +39,12 @@ import {
     isRight
 } from "@helios-lang/type-utils"
 import { UplcProgramV1, UplcProgramV2, UplcDataValue } from "@helios-lang/uplc"
+import { BasicUplcLogger } from "@helios-lang/uplc/node_modules/@helios-lang/compiler-utils/src/errors/BasicUplcLogger.js"
```
Comment (christianschmitz) [inline src/txbuilder/TxBuilder.js:42]: I hope it's ok for you that I comment on the remainder of this PR once we've last the *uplc* PR and we can clean lines like this up

Comment (rjharmon) [reply src/txbuilder/TxBuilder.js]: Oh, that's a mistake.  Sorry about that.

Comment (rjharmon) [reply src/txbuilder/TxBuilder.js]: was fixed in the later push

Comment (rjharmon) [reply src/txbuilder/TxBuilder.js]: the import was fixed in the later push, but the symbol isn't used here anymore.

**Comment by rjharmon** [comment]
okay, that should be better now

File: src/txbuilder/TxBuilder.js [hunk 2]
```diff
@@ -1820,42 +1870,92 @@ export class TxBuilder {
...
+        return profile
+    }
 
-            return redeemer
-        })
+    /**
+     * @private
+     * @returns {UplcLoggingI}
+     */
+    mkNullLogger() {
```
Comment (christianschmitz) [inline src/txbuilder/TxBuilder.js:1957]: can we inline this?

**Review by christianschmitz — APPROVED** [review]
excellent

**Pull request merged**